main.py

```diff
@@ -162,74 +285,113 @@ def execute():
             logger.debug('bypass mature content block')
             try:
                 WebDriverWait(browser, TIMEOUT).until(
-                    EC.visibility_of_element_located((By.XPATH, "//p[contains(text(),'mature content')]")))
+                    EC.visibility_of_element_located((
+                        By.XPATH,
+                        "//span[contains(text(),'mature content')]"
+                    ))
+                )
+
                 WebDriverWait(browser, TIMEOUT).until(
-                    EC.visibility_of_element_located((By.XPATH, "//span[contains(text(),'Continue')]"))).click()
+                    EC.visibility_of_element_located((
+                        By.XPATH,
+                        "//button[contains(text(),'Continue')]"
+                    ))
+                ).click()
+
             except TimeoutException:
                 logger.debug('no mature content block to bypass')
 
             logger.debug('find the purchase button')
             purchase_button = WebDriverWait(browser, TIMEOUT).until(
-                EC.visibility_of_element_located((By.XPATH, "//button[contains(@class,'Purchase')]"))
+                EC.visibility_of_element_located((
+                    By.XPATH,
+                    "//button[@data-testid='purchase-cta-button']"
+                ))
             )
 
             # name of the game
             logger.debug('extract game title')
-            name = browser.find_element_by_xpath("//h1[contains(@class,'NavigationVertical')]").text
+            name = browser.title
```

Comment:
This is no producing the same result, it's giving me "Get A Free Game Every Week | Epic Games Store"

Reply:
The one previously there will work if you change the xpath from h1 to h2

Reply:
ok, that's strange. I had opted to use the page title because sometimes it didn't read me the name of the game. He never gave me "Get A Free Game Every Week | Epic Games Store", neither in "debug mode" nor in "normal mode". I'll try again with your version